### DISCUSSION TITLE
Login reset

### DISCUSSION THREAD

**Discussion by bpodner** [datalust/seq-tickets#1780] [Help]
Trying to access a Seq log but the person whole setup the admin left the company. I tried the seq auth --basic script with nothing actually working. Any ideas?

**Comment by nblumhardt** [comment]
Hi! On new Seq versions, it's `seq auth reset` rather than `seq auth --basic` - let me know if this helps.

Otherwise, sending some info to support@ with the output of the reset attempt should let us spot what's going wrong.

Hope this helps,
Nick
